### PR TITLE
Fixed virtualization detection on Windows

### PR DESCRIPTION
**Pull request by kmazurek**
Resolves: #4024

Adds handling of a scenario where Hyper-V is installed, but has been
manually disabled by the user. Additionally, this removes the call to
WMI, since when Hyper-V is on it reports hardware virtualization as disabled.

### CODE REVIEW

**Review by Wiezzel — COMMENTED** [review]
Is the new version intended to work for Hyper-V only? What about DockerToolbox support?

**Comment by kmazurek** [comment]
@Wiezzel I did not take Docker Toolbox into account, but that's also true for the previous version of this script. My understanding is that the script should return true if either Hyper-V or Docker Toolbox is available. Is that correct?

**Comment by Wiezzel** [comment]
According to my knowledge the new version of the script will return `False` even if DockerToolbox is installed and virtualization enabled in BIOS.

**Comment by kmazurek** [comment]
Ok, understood - let me bring back that check.

**Review by Wiezzel — APPROVED** [review]
Seems OK now.

File: golem/docker/hypervisor/hyperv.py [hunk 1]
```diff
@@ -185,10 +185,16 @@ def start_vm(self, name: Optional[str] = None) -> None:
 
     @classmethod
     def is_available(cls) -> bool:
-        command = "@(Get-Module -ListAvailable hyper-v).Name | Get-Unique"
         try:
-            output = run_powershell(command=command)
-            return output == "Hyper-V"
+            output = run_powershell(
+                script=os.path.join(
```
Comment (Wiezzel) [inline golem/docker/hypervisor/hyperv.py]: Please, extract this path into a constant like `GET_VSWITCH_SCRIPT_PATH `.

**Review by Wiezzel — CHANGES_REQUESTED** [review]

**Review by Wiezzel — APPROVED** [review]
Alright now.

**Pull request merged**